src/main/java/com/uid2/admin/vertx/service/SharingService.java

```diff
@@ -150,6 +161,64 @@ private void handleSetKeyset(RoutingContext rc) {
         }
     }
 
+    // Method to check if one set contains any values from another set
+    private static <T> boolean containsAny(Set<T> set1, Set<T> set2) {
+        for (T element : set2) {
```

Comment:
is this method necessary? https://stackoverflow.com/questions/8708542/something-like-contains-any-for-java-set